data/json/items/gun/36paper.json

```diff
@@ -1,29 +1,31 @@
+Cataclysm web linting tool
```

Comment:
Well it's nice to see it getting some use but I think you got a bit carried away with the pasting there